scripts/create_video_from_file.py

```diff
@@ -49,13 +49,20 @@ def create_video_from_file(repository_path, file_path, branch, output, tmp_dir):
 
     images_paths = []
 
-    for index, file_content in enumerate(tqdm(file_generated_history, desc='Generating images...')):
+    for index, file_content in enumerate(tqdm(file_generated_history, desc='Generating images')):
         html_file_path = os.path.join(tmp_dir, f'{file_name_without_ext}_{index}.html')
         css_file_path = os.path.join(tmp_dir, f'{file_name_without_ext}_{index}.css')
         image_file_path = os.path.join(tmp_dir, f'{file_name_without_ext}_{index}.png')
 
         code_to_html(file_content, file_path, html_file_path, css_file_path)
-        imgkit.from_file(str(html_file_path), str(image_file_path), options={'quiet': 1})
+        imgkit.from_file(
+            str(html_file_path),
+            str(image_file_path),
+            options={
+                'quiet': 1,
+                'width': 1,
```

Comment:
I can't find reference for this parameter.

Reply:
Documented here: https://www.mankier.com/1/wkhtmltoimage#--width (`imgkit` uses `wkhtmltoimage` command).
It's called `width` but as I understood, this parameter is handled as a `min_width`, while `--disable-smart-width` parameter is not set. That's why I put a value of `1` (`0` is not accepted), so that it outputs an image with the smallest width still containing all html content.